migrations/versions/43e8c59339af_.py

```diff
@@ -0,0 +1,25 @@
+"""Changing field type of gender from `Text` to `Select`
+
+Revision ID: 43e8c59337af
+Revises: 43e8c59337ae
+Create Date: 2019-06-27 20:31:58.92665
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+ # revision identifiers, used by Alembic.
+revision = '6f7b6fad3f54'
+down_revision = '6f7b6fad3f53'
+
+
+def upgrade():
+    op.execute("UPDATE custom_forms SET type = 'select' where field_identifier = 'gender';", execution_options=None)
```

Comment:
make it more specific, only the form type 'attendee' is being refactored.

Reply:
@CosmicCoder96 How do I select the `form_type` to `attendee`, Will adding an `and form = 'attendee'` be suffice or there is some other way?